packages/integrations/mdx/test/mdx-get-static-paths.test.js

```diff
@@ -23,7 +23,7 @@ describe('getStaticPaths', () => {
 		const $ = cheerio.load(html);
 		expect($('p').text()).to.equal('First mdx file');
 		expect($('#one').text()).to.equal('hello', 'Frontmatter included');
-		expect($('#url').text()).to.equal('/src/content/1.mdx', 'url is included');
+		expect($('#url').text()).to.equal('src/content/1.mdx', 'url is included');
```

Comment:
The `base` handling in the tests previously patches the config, causing the leading slash. In a normal Astro app, there shouldn't be a leading slash.